tests/integration/types_test.ts

```diff
@@ -0,0 +1,303 @@
+import { assert } from "../../src/dev_deps.ts";
+import { DefineProperty, DefineType, Schema } from "../../src/mod.ts";
+import { CanBe } from "../../src/test_utils.ts";
+import { RuntimeType } from "../../src/types.ts";
+
+Deno.test("RuntimeType should provide a usable type from DefineType returned object", () => {
+  const customType = DefineType({
+    title: "Title",
+    description: "Description",
+    name: "Name",
+    type: Schema.types.boolean,
+  });
+
+  type Actual = RuntimeType<typeof customType>;
+
+  assert<CanBe<boolean, Actual>>(true);
```

Comment:
Nice! Good test, but one small change I would do here is, at least for this specific test case, I would replace `CanBe` with `IsExact`. `CanBe` tests set membership, whereas `IsExact` tests set equality. In this case, since the custom type.. type is `boolean`, we expect its runtime type to only be `boolean` - nothing else. So we want to test set equality.

I think that applies to most of the assertions in the tests you wrote.

For example, consider the following types, and see how `CanBe` vs. `IsExact` differ:

```typescript
type bigSet = boolean | string;
type smallSet = boolean;
assert<CanBe<boolean, Actual>>(true);
```

Reply:
Ive updated every test to use `IsExact` where possible 🥇